web/src/main/java/org/springframework/security/web/session/ContinueRequestSessionInformationExpiredStrategy.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2002-2016 the original author or authors.
```

Comment:
Can you make sure that the copyright notice includes the current year?

Reply:
Thanks! I changed